nengo/tests/options.py

```diff
@@ -0,0 +1,24 @@
+def pytest_addoption(parser):
```

Comment:
To make it importable for other backends without importing `conftest.py`. [See also.](https://github.com/nengo/nengo_ocl/pull/65)